src/components/widget_types/twitter.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div>
+        <v-card v-for="tweet in tweets.slice(0, numberTweets)" v-bind:key="tweet.text">
```

Comment:
DOM handlers might be slower than handling this in a method? Either way, I think it would look nicer if you add this as a computed method below. Something like:
computed : {
   tweets_sliced : function() {
      return this.tweets.slice(0,this.numberTweets);
   }
}
and then you just do a v-for="tweet in tweets_sliced" for this. Idk I think it would look cleaner, but does the same thing.

Reply:
good idea, made this change